libkbfs/interfaces.go

```diff
@@ -486,6 +486,15 @@ type KeyMetadata interface {
 		keyGen KeyGen, user keybase1.UID, key CryptPublicKey) (
 		TLFEphemeralPublicKey, EncryptedTLFCryptKeyClientHalf,
 		TLFCryptKeyServerHalfID, bool, error)
+
+	// AreHistoricTLFCryptKeysPerDeviceEncrypted returns whether or not historic
+	// keys are encrypted per-device or if they're symmetrically encrypted with
+	// the current generation's TLFCryptKey.
+	AreHistoricTLFCryptKeysPerDeviceEncrypted() bool
+
+	// GetTLFCryptKey attempts to symmetrically decrypt the key at the given
+	// generation using the current generation's TLFCryptKey.
+	GetTLFCryptKey(c cryptoPure, keyGen KeyGen, key TLFCryptKey) (TLFCryptKey, error)
```

Comment:
I think a better name is `Get{Old,Previous,Historic}TLFCryptKey`, with the `key` param being renamed to `{latest,current}Key`. The comment here is helpful, but the implementations won't have this comment.